README.md

```diff
@@ -49,7 +49,7 @@ The site should now be accessible on `localhost:4000` on the host, and can be mo
 
 ### Building for Deployment
```

Comment:
@kai33 Think of it as a preprocess method to reduce the number of requests for PPTLabs website. Since we merge local dependencies into one html file, the user don't have to request for respective js/css files anymore, thus saving precious time for the user :)